### DISCUSSION TITLE
Browser deletes cookies BEFORE cookie_expiration

### DISCUSSION THREAD

**Discussion by jakubmilata** [orestbida/cookieconsent#317] [Q&A]
I have noticed that my cookies are deleted BEFORE expiration. 
I have set expiration for 365days but I have to agree with cookie consent time to time on my own website!
I have never manually deleted cookies. 
It happens in google chrome on my PC. And I think that it happens even more frequently on my iphone. 
How can I prevent this?

**Comment by dombyte** [comment]
Have you checked your browser settings. You're Browser can automatically delete cookies.
